packages/botonic-react/src/webchat-app.jsx

```diff
@@ -49,7 +49,7 @@ export class WebchatApp {
 
   async onUserInput({ user, input }) {
     this.onMessage && this.onMessage(this, { from: 'user', message: input })
-    this.hubtypeService.postMessage(user, input)
+    return this.hubtypeService.postMessage(user, input)
```

Comment:
@marcrabat @ericmarcos can you review this, please?

Reply:
looks good to me